views/pages/editTask.ejs

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <%- include('../partials/head'); %>
+</head>
+
+<body class="container">
+
+  <header>
+    <%- include('../partials/header'); %>
+  </header>
+
+  <main>
+    <p>
+      <%= message %>
+    </p>
+    <form action=<%= "/tasks/update/" + task.id %> method="post">
+      <label for="name">Name:</label> <br>
+      <input name="name" value="<%=task.name %>"><br>
+      <label for="complete">Completed: </label>
+      <% if (task.completed) { %>
```

Comment:
Hm, is it possible to be a little more targeted here? I'm not familiar with ejs, but something like:

```
<input type="checkbox" value="{{ task.completed }}" />
```

Where we directly put the "task.completed" value into the `value` attribute?

Reply:
I am not sure this is our 1st session with ejs. This statement is done so we can check the checkbox as checked if its true and if false it wont be checked.

Most of this code was given to us in the assignment and John said he will be fixing some of it, However I